### PR TITLE
Add 'login.defaultRedirect'

### PR DESCRIPTION
**Pull request by juhangsin**

### CODE REVIEW

File: src/login/controller/login.ts [hunk 1]
```diff
@@ -73,8 +73,20 @@ class LoginController extends Controller {
...
+      ctx.response.headers.set('Location', '/');
+    }
+
+  }
 
+  isRedirect() {
+    if (getSetting('login.defaultRedirect') !== '/') {
+      return true;
+    }
+    return false;
```
Comment (evert) [inline src/login/controller/login.ts:89]:
```suggestion
    ctx.response.redirect(303, getSetting('login.defaultRedirect', '/'));
```

Comment (evert) [reply src/login/controller/login.ts:89]: I think this will work

File: src/server-settings.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ type Settings = {
   'webauthn.expectedOrigin': string,
   'webauthn.relyingPartyId': string,
   'webauthn.serviceName': string,
+  'login.defaultRedirect': '/' | string,
```
Comment (evert) [inline src/server-settings.ts:13]: just make this `string`. Not a bad idea but `/` is still just a string

**Review by evert — CHANGES_REQUESTED** [review]

File: src/login/controller/login.ts (hunk 1 above)
Comment (juhangsin) [reply src/login/controller/login.ts:89]: ahhhh yeah, i guess function has default params :( my bad

**Review by evert — APPROVED** [review]

**Pull request merged**